pubky-sdk/README.md

```diff
@@ -0,0 +1,302 @@
+# Pubky SDK
+
+Ergonomic building blocks for Pubky apps: one facade (`Pubky`) plus focused actors for sessions, storage API, signer helpers, and QR auth flow for keyless apps.
+
+Rust implementation of [Pubky](https://github.com/pubky/pubky-core) SDK.
+
+## Install
+
+```toml
+# Cargo.toml
+[dependencies]
+pubky = "0.x"            # this crate
+# Optional helpers used in examples:
+# pubky-testnet = "0.x"
+```
+
+## Quick start
+
+```rust no_run
+use pubky::prelude::*;
+
+# async fn run() -> pubky::Result<()> {
+
+let pubky = Pubky::new()?; // or Pubky::testnet() for local testnet.
+
+// 1) Create a new random key user and bound to a Signer
+let keypair = Keypair::random();
+let signer = pubky.signer(keypair);
+
+// 2) Sign up on a homeserver (identified by its public key)
+let homeserver = PublicKey::try_from("o4dksf...uyy").unwrap();
+let session = signer.signup(&homeserver, None).await?;
+
+// 3) Read/Write as the signed-in user
+session.storage().put("/pub/my-cool-app/hello.txt", "hello").await?;
+let body = session.storage().get("/pub/my-cool-app/hello.txt").await?.text().await?;
+assert_eq!(&body, "hello");
+
+// 4) Public read of another user’s file
+let txt = pubky.public_storage()
+  .get(format!("pubky{}/pub/my-cool-app/hello.txt", session.info().public_key()))
+  .await?
+  .text().await?;
+assert_eq!(txt, "hello");
+
+// 5) Keyless app flow (QR/deeplink)
+let caps = Capabilities::builder().write("/pub/example.com/").finish();
+let flow = pubky.start_auth_flow(&caps)?;
+println!("Scan to sign in: {}", flow.authorization_url());
+let app_session = flow.await_approval().await?;
+
+// 6) Optional (advanced): publish or resolve PKDNS (_pubky) records
+signer.pkdns().publish_homeserver_if_stale(None).await?;
+let resolved = signer.pkdns().get_homeserver().await;
+println!("Your current homeserver: {:?}", resolved);
+
+# Ok(()) }
+```
+
+## Mental model
+
+- `Pubky` - facade, always start here! Owns the transport and constructs actors.
+- `PubkySigner` - local key holder. Can `signup`, `signin`, approve QR auth, publish PKDNS.
+- `PubkySession` - authenticated “as me” handle. Exposes session-scoped storage.
+- `PublicStorage` - unauthenticated reads of others’ public data.
+- `Pkdns` - resolve/publish `_pubky` records.
+
+#### Transport:
+
+- **`PubkyHttpClient`** stateless transport: handles requests to pubky public-key hosts.
```

Comment:
```suggestion
- **`PubkyHttpClient`** - handles requests to pubky public-key hosts.
```

Addressing last remaining instance of https://github.com/pubky/pubky-core/pull/219#discussion_r2435068144